### PR TITLE
Add update and remove functions, and adding styles to the task cards

### PR DESCRIPTION
**Pull request by ivangonzalez224**
It's my To do list project and I added the following features:

- Added styles to task cards.
- Added events listeners to add, update and remove tasks.
- Added a module for remove and update functions.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
-// crea tasks list//
-createTask();
+
+// retrieve tasks data and display the task list
+window.onload = () => {
+  if (localStorage.getItem('dataTasks') !== 'undefined' || localStorage.getItem('dataTasks') !== 'null') {
+    dataTask = JSON.parse(localStorage.getItem('dataTasks'));
+    createTask(0, dataTask.length, dataTask);
+  }
+};
```
Comment (nahnahdev) [inline src/index.js]: - [ ] Great job so far @ivangonzalez224 there are just a few changes I will suggest you make and you'll be on your way to the next milestone :wink: 
- [ ] When I run your project, there was an error that was preventing any of the functionalities you were to implement in this milestone from working at all. I later found out that the initial error was coming from this part of your code so kindly fix this so your app can run seamlessly and work as expected without any errors :+1: 
- [ ] I tried to fix the error so I could test if your functionalities were working so I edited this part of your code to look something like this after which the error disappeared :
```javascript
window.onload = () => {
  const storedData = localStorage.getItem('dataTasks');
  if (storedData !== null) {
    dataTask = JSON.parse(storedData);
    createTask(0, dataTask.length, dataTask);
  }
};
```
- Error Message
image

File: src/modules/addTask.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+let dataTaskAdd = [];
+// add a new task
+const addTask = () => {
+  dataTaskAdd = JSON.parse(localStorage.getItem('dataTasks'));
+  const lengthList = dataTaskAdd.length + 1;
+  dataTaskAdd.push({ description: addTaskInput.value, completed: false, index: lengthList });
+  addTaskInput.value = '';
+  createTask(lengthList - 1, lengthList, dataTaskAdd);
+  localStorage.setItem('dataTasks', JSON.stringify(dataTaskAdd));
+};
```
Comment (nahnahdev) [inline src/modules/addTask.js]: - [ ] After being able to solve the first error, another error popped up when I tried to add a `list item` so I will suggest you fix that as well :100: So users are able to add and remove items in your app :+1: 
- [ ] However, I modified this part of your code to look something like this below and afterward the app was working fine. So kindly fix that as well :
```javascript
const addTask = () => {
  const storedData = localStorage.getItem('dataTasks');
  if (storedData) {
    dataTaskAdd = JSON.parse(storedData);
  } else {
    dataTaskAdd = [];
  }
  ```
- Error Message
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @ivangonzalez224 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- Linters well setup and passing :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- Most project requirements met :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @ivangonzalez224, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**